cds_ils/migrator/utils.py

```diff
@@ -129,7 +130,32 @@ def clean_item_record(record):
     clean_item_status(record)
     clean_description_field(record)
     record["shelf"] = record["location"]
-    record["medium"] = "NOT_SPECIFIED"
+    record["medium"] = "PAPER"
     del record["location"]
     del record["id_bibrec"]
     del record["id_crcLIBRARY"]
+
+
+def clean_created_by_field(record):
+    """Clean the created by field for documents."""
+    if "_email" in record["created_by"]:
+        patron = User.query.filter_by(
+            email=record["created_by"]["_email"]
+        ).one_or_none()
+        if patron:
+            patron_pid = patron.get_id()
+            record["created_by"]["type"] = "user_id"
+            record["created_by"]["value"] = patron_pid
+        else:
+            record["created_by"]["type"] = "user_id"
+            record["created_by"]["value"] = "anonymous"
+        del record["created_by"]["_email"]
+    elif record["created_by"]["type"] in [
```

Comment:
Isn't it better to remove the extra check?
```suggestion
```
I think it might be better to always have a value, otherwise editing of migrated records might fail (not sure about this)?

Reply:
@KonstantinaStoikou lets have a call about it because it does not cover all the possible cases we talked about